crates/pop-parachains/src/call/mod.rs

```diff
@@ -92,11 +92,16 @@ mod tests {
 
 	#[tokio::test]
 	async fn construct_extrinsic_works() -> Result<()> {
+		let client = set_up_client("wss://rpc1.paseo.popnetwork.xyz").await?;
+		let pallets = parse_chain_metadata(&client).await?;
```

Comment:
For future - it would probably be better to place this behind a lazy static or similar so that it is done once per test **run** rather than for every test, reducing the number of metadata parse requests.

Reply:
This is great feedback for improving the tests. Noting it down for a future task.